tests/test_rewrite_paragraph.py

```
@@ -0,0 +1,702 @@
+
+        vis = doc.get_visible_text()
+        assert "The cat and the dog and a bird" in vis
+
+        revisions = doc.list_revisions()
+        del_texts = [r.text for r in revisions if r.type == "deletion"]
+        ins_texts = [r.text for r in revisions if r.type == "insertion"]
+        assert any("the" in t.lower() for t in del_texts)
+        assert any("a" == t.strip() for t in ins_texts)
+
+
+def _make_doc_with_paragraphs(paragraphs: list[str]):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_make_doc_with_paragraphs`

(ANN202)